pyproject.toml

```diff
@@ -12,7 +12,7 @@ dependencies = [
   "celery",
   "dandi>=0.60.0",  # minimal version is also provided in API /info
   # Pin dandischema to exact version to make explicit which schema version is being used
-  "dandischema==0.11.1",  # schema version 0.6.10
+  "dandischema @ git+https://github.com/dandi/dandi-schema.git@devendorize",
```

Comment:
This is temporary, correct? We'll want to set this back to the necessary dandischema version before this is merged.